app/src/main/java/com/example/playlistmaker/player/ui/AudioPlayerViewModel.kt

```diff
@@ -0,0 +1,103 @@
+package com.example.playlistmaker.player.ui
+
+import android.app.Application
+import android.os.Handler
+import android.os.Looper
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProvider.AndroidViewModelFactory.Companion.APPLICATION_KEY
+import androidx.lifecycle.viewmodel.initializer
+import androidx.lifecycle.viewmodel.viewModelFactory
+import com.example.playlistmaker.R
+import com.example.playlistmaker.creator.Creator
+import com.example.playlistmaker.player.data.mediaplayer.PlayerStatus
+import com.example.playlistmaker.player.ui.models.PlayerState
+import com.example.playlistmaker.search.ui.TrackItemAdapter
+
+
+class AudioPlayerViewModel(application: Application) : AndroidViewModel(application) {
+    private var mediaPlayer = Creator.provideMediaPlayerInteractor()
+    private var track = TrackItemAdapter.track
```

Comment:
```suggestion
    private val mediaPlayer = Creator.provideMediaPlayerInteractor()
    private val track = TrackItemAdapter.track
```
Обе переменные не должны изменяться в рамках жизненного цикла этой вьюмодели